Presentation.Web/Tests/01_Parallel/it-system/LocalAdmin.System.e2e.spec.ts

```diff
@@ -1,41 +0,0 @@
-﻿import Login = require("../../Helpers/LoginHelper");
-import ItSystemOverviewPo = require("../../PageObjects/it-system/Usage/ItSystemUsageOverview.po");
```

Comment:
I think it is a mistake to remove this test. We could add a test to validate that the dropdown is present only when local admin is logged in